src/referencePackages/Directory.Build.targets

```diff
@@ -21,23 +33,24 @@
   <!-- Customization extension point -->
   <Import Project="$(CustomizationsPropsPath)" Condition="Exists('$(CustomizationsPropsPath)')" />
 
-  <!--
-    ### Targeting Packs section ###
-    Keep in sync with available targeting packs under src/targetPacks/ILsrc.
-  -->
-
   <PropertyGroup>
+    <!-- Avoid transitive framework reference downloads to minimize the number of targeting packs and prebuilts. -->
     <DisableTransitiveFrameworkReferenceDownloads>true</DisableTransitiveFrameworkReferenceDownloads>
     <!-- The SDK already sets the NETStandardImplicitPackageVersion and we don't expect it to change anymore. Hence, we don't encode it here. -->
   </PropertyGroup>
 
+  <!-- Keep in sync with available targeting packs under src/targetPacks/ILsrc. -->
   <ItemGroup>
     <KnownFrameworkReference Update="@(KnownFrameworkReference->WithMetadataValue('Identity', 'Microsoft.NETCore.App'))">
       <TargetingPackVersion Condition="'%(TargetFramework)' == 'netcoreapp3.0'">3.0.0</TargetingPackVersion>
       <TargetingPackVersion Condition="'%(TargetFramework)' == 'netcoreapp3.1'">3.1.0</TargetingPackVersion>
       <TargetingPackVersion Condition="$([System.Text.RegularExpressions.Regex]::IsMatch('%(TargetFramework)', '^net\d+\.0$'))">$([System.Text.RegularExpressions.Regex]::Match('%(TargetFramework)', '\d+').Value).0.0</TargetingPackVersion>
     </KnownFrameworkReference>
 
+    <KnownFrameworkReference Update="@(KnownFrameworkReference->WithMetadataValue('Identity', 'Microsoft.AspNetCore.App'))">
+      <TargetingPackVersion Condition="'%(TargetFramework)' == 'net6.0'">6.0.2</TargetingPackVersion>
+    </KnownFrameworkReference>
```

Comment:
We also have a 6.0.2 version of Microsoft.NetCore.App.Ref. Does that need an entry like this as well?